docs/source/installing-brave.rst

```diff
@@ -180,3 +180,15 @@ Centos/RHel
     enabled=1
     EOF
     sudo yum install brave-browser-dev
+
+
+Opensuse
+----------
+::
+    sudo rpm --import  https://brave-browser-rpm-dev.s3.brave.com/brave-core-nightly.asc
+    
+    zypper ar -f https://brave-browser-rpm-dev.s3.brave.com/x86_64/ brave
+    
+    zypper update
+    
+    zypper install brave-browser-de
```

Comment:
missing a `v` at the end to be `brave-browser-dev`?

Reply:
This seems a bug from rebasing